src/server/game/Server/WorldSession.h

```diff
@@ -81,6 +81,10 @@ namespace WorldPackets
         class Hello;
         class TrainerBuySpell;
     }
+    namespace Misc
+    {
+        class Weather;
```

Comment:
still not a proper cherry-pick - this line does not exist in master and forward declaring in WorldSession is not needed for SMSG packets

Reply:
I'll just do it myself later today